.github/workflows/cli-foundry-test.yml

```
@@ -0,0 +1,47 @@
+name: foundry-test
+
+on:
+  pull_request:
+    paths:
+      - target_chains/ethereum/sdk/stylus/**                                                                 **
```

```suggestion
      - target_chains/ethereum/sdk/stylus/**
```
There are some stray symbols in all the workflow files. Could you please remove them and then add another commit that changes files in the stylus directory?